code/modules/surgery/bodyparts/bodyparts.dm

```diff
@@ -124,10 +131,9 @@
 	brute = max(brute * dmg_mlt, 0)
 	burn = max(burn * dmg_mlt, 0)
 	stamina = max(stamina * dmg_mlt, 0)
-	if(status == BODYPART_ROBOTIC) //This makes robolimbs not damageable by chems and makes it stronger
-		brute = max(0, brute - 5)
-		burn = max(0, burn - 4)
-		//No stamina scaling.. for now..
+	brute = max(0, brute - brute_reduction)
+	burn = max(0, burn - burn_reduction)
+	//No stamina scaling.. for now..
```

Comment:
This comment about the stamina scaling no longer make sense with robotic damage reduction being moved to the limbs.

Reply:
Oh yeah you're right